daemon/src/model/cfd.rs

```diff
@@ -1280,15 +1284,15 @@ pub enum CfdStateChangeEvent {
     OracleAttestation(Attestation),
     CetSent,
     /// Settlement proposal was signed by the taker and sent to the maker
-    ProposalSigned(TimestampedTransaction),
+    ProposalSigned(CollaborativeSettlement),
     /// Maker signed and finalized the close transaction with both signatures
-    CloseSent(TimestampedTransaction),
+    CloseSent(CollaborativeSettlement),
 }
 
 /// Returns the Profit/Loss (P/L) as Bitcoin. Losses are capped by the provided margin
 fn calculate_profit(
     initial_price: Usd,
-    current_price: Usd,
```

Comment:
isn't this function also used with the current price to present in the table when the Cfd is still Open? 

maybe start/end price?

Reply:
not sure about the names, happy to rename in follow up 😅